eval/README.md

```diff
@@ -0,0 +1,63 @@
+## Introduction:
```

Comment:
Do you want to rename this to readme_trajectory.md or something @Anusha-1209

Reply:
@Anusha-1209  pls update once your readme changes are done as well, thanks.
also link to your readme from the main readme.